projects/velvet-capital/index.js

```diff
@@ -0,0 +1,80 @@
+const sdk = require("@defillama/sdk");
+
+async function tvl(){
+
+    var responseData = {
+        indexes: ["0x3527069C603b7d818aA0D3c15Bd4d8d5914aD66a","0xCE5a3270e5904260B7E4F4CC6e105401ce08788D","0xf4196172911E47074F8f601339438e4a50075153","0x3DC60aeC556FCC31E0f86be698550f439e367ECb","0x5A4957F97C2Ac01793C2aa1d06c6C9f6C20592b8","0x95bD9c6d2964A560B84788d55a8a135343E1129A","0x6d494a17f0D6001B20939e2C0dbF2b2Ea4F86393","0x7841988cA1A133eb30a156562d401B28b5cFaAbd","0xb4829a14A8F1097592D05e88F5c008baAcBF1077"],
```

Comment:
can this list be fetched on chain somehow?

Reply:
I have refactored your code a bit here, hopefully didnt messup the logic: https://github.com/DefiLlama/DefiLlama-Adapters/pull/5261